tests/numpy/test_misc.py

```diff
@@ -113,20 +113,55 @@ def test_all(nps_app_inst):
     assert nps.all(true_float).get() == np.all(np.array([[1.0, 2.0, 3.0], [1.0, 2.0, 3.0]]))
     assert nps.all(false_float).get() == np.all(np.array([[1.0, 2.0, 0.0], [1.0, 2.0, 3.0]]))
 
+    assert nps.alltrue(true_int)
+    assert not nps.alltrue(false_int)
+    assert nps.alltrue(true_bool)
+    assert not nps.alltrue(false_bool)
+    assert nps.alltrue(true_float).get() == \
+           np.alltrue(np.array([[1.0, 2.0, 3.0], [1.0, 2.0, 3.0]]))
+    assert nps.alltrue(false_float).get() == \
+           np.alltrue(np.array([[1.0, 2.0, 0.0], [1.0, 2.0, 3.0]]))
+
+    false_int = nps.array([[0, 0, 0], [0, 0, 0]]).reshape(block_shape=(2, 2))
+    false_bool = nps.array([False, False])
+    false_float = nps.array([[0.0, 0.0, 0.0], [0.0, 0.0, 0.0]]).reshape(block_shape=(2, 2))
+
+    assert nps.any(true_int)
+    assert not nps.any(false_int)
+    assert nps.any(true_bool)
+    assert not nps.any(false_bool)
+    assert nps.any(true_float).get() == np.any(np.array([[1.0, 0.0, 0.0], [0.0, 0.0, 0.0]]))
+    assert nps.any(false_float).get() == np.any(np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 0.0]]))
+
     assert nps.all(true_int).dtype is np.bool
```

Comment:
Can you put `true_int`, `false_int`, etc. in a list and run these assertions in a loop?

Reply:
Yep, and I'll keep doing this from now on whenever possible.